docker-compose-fullstack.yml

```diff
@@ -0,0 +1,43 @@
+version: '3'
+
+services:
+  postgres:
+    image: postgres:9.6-alpine
+    ports:
+      - "5432:5432"
+    volumes:
+      - pg_data:/var/lib/postgresql
+
+  redis:
+    image: redis:3-alpine
+    ports:
+      - "6379:6379"
+
+  website:
```

Comment:
This should declare the dependencies, instead of assuming they are there. It allows us to `docker-compose -f docker-compose-fullstack.yml up website` or run one-off commands in that container as necessary

```
website:
  image: pyslackers/website
  build: .
  ports:
    - "8000:8000"
  depends_on:
    - postgres
    - redis
  environment:
    ...
```